tests/unit/test_irc.py

```diff
@@ -126,22 +116,23 @@ def test_prepare_does_not_copy_files_if_already_exist(irc_bridge_service, mocker
     systemd.daemon_reload, and systemd.service_enable methods were called exactly once.
     """
     mock_install_snap_package = mocker.patch.object(irc_bridge_service, "_install_snap_package")
```

Comment:
I think we should review some of these tests, as I think they are too coupled to the implementation. Not sure how we do it in other charms.

Reply:
In the case of test_prepare_does_not_copy_files_if_already_exist, I opted to remove the test because it was essentially redundant. IMHO is just testing if Path.exists works.